src/main/java/frc/robot/OperatorInterface.java

```diff
@@ -20,89 +17,82 @@
     private JoystickButtonManager operatorPanelManager;
     private SubsystemManager subsystemManager;
     private DriveSubsystem drive;
-    private CommandGroupFactory commandFactory;
+    private CommandFactory commandFactory;
 
     public OperatorInterface(final SubsystemManager subMan) {
         this.subsystemManager = subMan;
-        commandFactory = new CommandGroupFactory((subsystemManager));
+        commandFactory = new CommandFactory(subsystemManager);
```

Comment:
Wasn't there a discussion of just putting the commands in the subsystem manager that way they were all together and we did not need to have a seperate class that literally takes a subsystem manager.

Reply:
i moved all of the command creation into CommandFactory.  moving it all into subsystem manager makes that class do two things, and makes it really big.

subsystem manager should have all of the subsystems, but CommandFactory should hvae only the ones it needs, which will end up being fewer than all of them.